javascript/javascript3/week3/ShoppingCartUsingClasses/main.js

```diff
@@ -0,0 +1,115 @@
+//Shopping Cart Homework js3/week3
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+
+  convertToCurrency(currency) {
+    if (currency.toLowerCase().includes("dollar")) {
+      return (this.price / 6.27).toFixed(2);
+    } else if (currency.toLowerCase().includes("euro")) {
+      return (this.price / 7.44).toFixed(2);
+    } else if (currency.toLowerCase().includes("pound")) {
+      return (this.price / 8.67).toFixed(2);
+    } else {
+      return this.price;
+    }
+  }
+}
+
+class ShoppingCart {
+  constructor() {
+    this.products = [];
+  }
+
+  addProduct(product) {
+    this.products.push(product);
+  }
+
+  removeProduct(product) {
+    const deleteItem = this.products.filter(
+      (item) => product.name !== item.name
+    );
+    this.products = deleteItem;
+  }
+  searchProduct() {
+    const input = document.querySelector("#search-box").value;
+    if (input) {
+      const found = this.products.filter((item) => item.name === input);
+      console.log(`Found item is ${found}`);
```

Comment:
The `filter` method returns an array, so when you log the results it wil print `Found item is [object object]`

Reply:
i tried to resolve this problem , but i actually did not get any idea to search the product based on the user input, can you please suggest some solution for this , i just want to search the input given by the user, if it is found then log its found else its not found. thanks